client/src/components/CheckBox/CheckBox.scss

```diff
@@ -9,3 +9,12 @@
   flex-shrink: 0;
   flex-grow: 0;
 }
+
+// in CheckBox.tsx (1)
+.span-hover {
```

Comment:
To match the existing class names used in `CheckBox`, and use BEM, I'd suggest renaming this class as `checkbox-span--interactive`. Should avoid generic names outside of the global style sheet modules (to avoid accidentally reusing a name in multiple places).

Reply:
Done! Also I apologize for the comments, those were just for tracking purposes, I'll delete them!

Reply:
No worries!